TrackletAlgorithm/TrackletCalculator.h

```diff
@@ -78,18 +79,18 @@ namespace TC {
   static const ap_uint<9> z0cut_L1L2 = 15.0 / kz0 + 1.0e-1;
   static const ap_uint<9> z0cut = 1.5 * 15.0 / kz0 + 1.0e-1;
   static const Types::rmean RMEAN[6] = {
-    25.137 / kr + 1.0e-1,
```

Comment:
do these belong in the constants file?

Reply:
Yes, I think so. I've moved them to Constants.h.